r/rules_rust/rules_rust_0.42.1.patch

```diff
@@ -0,0 +1,306 @@
+diff --git a/cargo/cargo_build_script_runner/bin.rs b/cargo/cargo_build_script_runner/bin.rs
+index 8730c809..54b8aa71 100644
+--- a/cargo/cargo_build_script_runner/bin.rs
++++ b/cargo/cargo_build_script_runner/bin.rs
```

Comment:
so @vinodk99 will this patch changes work for older release version also or we should add different patch file per release version?

Reply:
@nikhil-kalbande, yes the patch file is working for older versions also